test/tests/divertor_monoblock/divertor_monoblock.i

```diff
@@ -16,6 +16,18 @@
 
 tungsten_atomic_density = ${units 6.338e28 m^-3}
 
+plasma_ramp_time = 100.0
+plasma_ss_end_time = 500.0
+plasma_ramp_down_time = 600.0
+plasma_cycle_time = 1600.0
```

Comment:
Minor comment: what if we defines these as durations and then add them to get the times - makes it easier to change just one of them and ensure the rest are consistent

Reply:
I agree with this suggestion.